### PR TITLE
OpenMusic.OpenMusic version 7.0

### PR DESCRIPTION
**Pull request by charlesneimog**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/69688)

### CODE REVIEW

File: manifests/o/OpenMusic/OpenMusic/7.0/OpenMusic.OpenMusic.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OpenMusic.OpenMusic
+PackageVersion: 7.0
+PackageLocale: en-US
+Publisher: Ircam
```
Comment (ItzLevvie) [inline manifests/o/OpenMusic/OpenMusic/7.0/OpenMusic.OpenMusic.locale.en-US.yaml:7]:
```suggestion
Publisher: IRCAM
```
This must match Control Panel.

File: manifests/o/OpenMusic/OpenMusic/7.0/OpenMusic.OpenMusic.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OpenMusic.OpenMusic
+PackageVersion: 7.0
+PackageLocale: en-US
+Publisher: Ircam
+PackageName: OM 7.0
```
Comment (ItzLevvie) [inline manifests/o/OpenMusic/OpenMusic/7.0/OpenMusic.OpenMusic.locale.en-US.yaml:8]:
```suggestion
PackageName: OM
```

**Comment by charlesneimog** [comment]
Here is working!! I do not understand!

**Comment by Trenly** [comment]
You will need to add installer switches to suppress the reboot
image

**Review by Trenly — CHANGES_REQUESTED** [review]
See above

**Pull request closed without merging**